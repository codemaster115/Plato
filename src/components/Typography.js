import React from 'react';
import Typography from '@material-ui/core/Typography';
import styled from 'styled-components';

export const StyledTypography = styled(Typography)`
    fontFamily: "Source Sans Pro" !important;
    font-size: 24px !important;
    font-weight: bold !important;
    color: rgb(82, 95, 127) !important;
    margin: 15px !important;
    text-align: center;
`;

class TypographyComponent extends React.Component {
  render() {
    return (
      <StyledTypography component="h1" variant="h4">
        { this.props.typography }
      </StyledTypography>
    );
  }
}

export default TypographyComponent;
