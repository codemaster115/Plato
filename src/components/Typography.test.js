import React from 'react';
import Enzyme, { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-styled-components';
import Typography from './Typography';

Enzyme.configure({ adapter: new Adapter() });

test('Compare snapshots', () => {
  const Component = renderer.create(<Typography />);

  const Tree = Component.toJSON();
  expect(Tree).toMatchSnapshot();
});
test('Renders without crashing', () => {
  mount(<Typography />);
});