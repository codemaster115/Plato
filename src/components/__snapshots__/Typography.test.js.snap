// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compare snapshots 1`] = `
.c0 {
  fontFamily: "Source Sans Pro" !important;
  font-size: 24px !important;
  font-weight: bold !important;
  color: rgb(82,95,127) !important;
  margin: 15px !important;
  text-align: center;
}

<h1
  className="MuiTypography-root-1 MuiTypography-h4-16 c0"
/>
`;
